addons/binding/org.openhab.binding.lutron/src/main/java/org/openhab/binding/lutron/handler/PicoKeypadHandler.java

```diff
@@ -0,0 +1,115 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.lutron.handler;
+
+import java.util.Arrays;
+
+import org.eclipse.smarthome.core.thing.Thing;
+import org.openhab.binding.lutron.internal.KeypadComponent;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Handler responsible for communicating with Lutron Pico keypads
+ *
+ * @author Bob Adair - Initial contribution
+ */
+public class PicoKeypadHandler extends BaseKeypadHandler {
+
+    private static enum COMPONENT implements KeypadComponent {
+        // Buttons for 2B, 2BRL, 3B, and 3BRL models
+        BUTTON1(2, "button1", "Button 1"),
+        BUTTON2(3, "button2", "Button 2"),
+        BUTTON3(4, "button3", "Button 3"),
+
+        RAISE(5, "buttonraise", "Raise Button"),
+        LOWER(6, "buttonlower", "Lower Button"),
+
+        // Buttons for PJ2-4B model
+        BUTTON01(8, "button01", "Button 1"),
```

Comment:
Maybe rename these to something like PJ2_4B_BUTTON1, etc.?  Seems a little confusing to have BUTTON1 and BUTTON01.

Reply:
Yeah, I guess that was a bit confusing. I renamed them to BUTTON1_4B and so on.